### DISCUSSION TITLE
Problem with installing software section

### DISCUSSION THREAD

**Discussion by scemmons** [biostars/biostar-handbook-issues#199] [Q&A] [answered]
Hi, I am brand new to this so apologies if I'm not understanding correctly. I followed the instructions in the book on installing the windows subsystem. The book states that the Ubuntu from the Windows store will save files to:
C:\Users\<Windows UserName>\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState\rootfs\home\<Ubuntu UserName>

However, when i typed in "explorer.exe ." mine was located on a network drive? Here is the path it took me to:
Network>wsl$>Ununtu>home><UbuntuUserName>
Which shows up as:
\\wsl$\Ubuntu\home\<UbuntuUserName>
when you click on the file explorer bar at the top.

I wasn't sure if this was an issue until I got to the "Run a realistic analysis section" and tried to run this line:
`make vcf`

which gave me this error:
`$ make vcf ACC=NC_045512  SRR=SRR17054510  CPU=4
mkdir -p refs
bio fetch NC_045512 -format gff > refs/NC_045512.gff
mkdir -p refs
bio fetch NC_045512 -format fasta > refs/NC_045512.fa
samtools faidx refs/NC_045512.fa
bwa index refs/NC_045512.fa
[bwa_index] Pack FASTA... 0.00 sec
[bwa_index] Construct BWT for the packed sequence...
[bwa_index] 0.00 seconds elapse.
[bwa_index] Update BWT... 0.00 sec
[bwa_index] Pack forward-only FASTA... 0.02 sec
[bwa_index] Construct SA from BWT and Occ... 0.00 sec
[main] Version: 0.7.17-r1188
[main] CMD: bwa index refs/NC_045512.fa
[main] Real time: 0.032 sec; CPU: 0.016 sec
echo refs/NC_045512.fa.amb
refs/NC_045512.fa.amb
mkdir -p reads
fastq-dump -F -X 10000 --split-files -O reads SRR17054510
Failed to call external services.
make: *** [Makefile:110: reads/SRR17054510_1.fastq] Error 64
(bioinfo)`

Looking at another post on here who had a similar issue (https://github.com/biostars/biostar-handbook-issues/discussions/184), the solution was suggested to be internet access (I am currently connected to the internet). However, I'm wondering if the installation being located on the network drive is causing this issue? I'm also curious how the installation can even be located on the network drive and not the location suggested by the book?

How can I resolve this issue, and will it cause me further issues as I progress through the book?
Thanks.

**Comment by ialbert** [comment, marked as answer]
The network mounting is fine, this appears to be a problem with how fastq-dump works

https://github.com/biostars/biostar-handbook-issues/discussions/196

another reader reported this issue I will edit the book to include the troubleshooting instruction:

> had to download latest ubuntu version of sratoolkit from ncbi @
> http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-ubuntu64.tar.gz

> I unpacked and put in home/myusername/bin folder created for doctor.py
> and added the sra's bin to the path in .bashrc

    export PATH=~/bin/sratoolkit.2.11.2-ubuntu64/bin:$PATH

**Comment by ialbert** [comment]
Made a new post here:

[Problems with fastq-dump (Failed to call external services)][1]

[1]: https://github.com/biostars/biostar-handbook-issues/discussions/200

I will attempt a more automated fix in the near future.

**Reply by scemmons** [reply]
Great, that worked! Thanks so much.
